lib/internal/modules/cjs/loader.js

```diff
@@ -1648,7 +1648,7 @@ function loadTS(module, filename) {
 function reconstructErrorStack(err, parentPath, parentSource) {
   const errLine = StringPrototypeSplit(
     StringPrototypeSlice(err.stack, StringPrototypeIndexOf(
-      err.stack, '    at ')), '\n', 1)[0];
+      err.stack, '    at ')), '\n', 2)[1];
```

Comment:
I don't think this is the correct way to solve it, but I couldn't find time to get a better solution

Reply:
Would you recommend closing? I just figured out the Window thing—for some reason, some newlines have two `\n` in a row.

But I don't know if it's worth dealing with it. Maybe it's better to let somebody else fix it and see how it could be done.

Reply:
Actually, I got something new https://github.com/nodejs/node/pull/55561/files#diff-5b5902273122e094ff474fda358605ffa45a4a58b51cd0bf4c1acb93779df142R1678